doc/buildAndProgram.md

```diff
@@ -48,7 +48,7 @@ CMake configures the project according to variables you specify the command line
 #### (\*) Note about **CMAKE_BUILD_TYPE**
 By default, this variable is set to *Release*. It compiles the code with size and speed optimizations. We use this value for all the binaries we publish when we [release](https://github.com/InfiniTimeOrg/InfiniTime/releases) new versions of InfiniTime.
 
-The *Debug* mode disables all optimizations, which makes the code easier to debug. However, the binary size will likely be too big to fit in the internal flash memory. If you want to build and debug a *Debug* binary, you'll need to disable some parts of the code. For example, the icons for the **Navigation** app use a lot of memory space. You can comment the content of `m_iconMap` in the [Navigation](https://github.com/InfiniTimeOrg/InfiniTime/blob/main/src/displayapp/screens/Navigation.h#L148) application to free some memory.
```

Comment:
Shouldn't the example of the navigation app be removed or changed? Since icons no longer reside in the main binary, removing them won't free up binary space?

If this example is being kept, the link URL needs to be updated

Reply:
Yes, you are right, removing `iconMap` won't free much memory now. I'll remove the example.

Reply:
I've just edited that part of the doc and provided an alternative way to build (parts of) the firmware in debug mode.

Reply:
Looks great :+1: